client/my-sites/site-settings/test/form-analytics.jsx

```diff
@@ -1,4 +1,7 @@
-/** @format */
+/**
+ * @format
+ * @jest-environment jsdom
```

Comment:
I think this is required in #24987 to avoid breaking the one test that keeps failing.
Mind removing this from here and adding it there?